arrangor/src/main/kotlin/no/nav/amt/tiltak/arrangor/ArrangorServiceImpl.kt

```diff
@@ -43,11 +44,20 @@ class ArrangorServiceImpl(
 		return arrangorer
 	}
 
-	override fun getOrCreateArrangor(virksomhetsnummer: String): Arrangor {
-		val maybeArrangor = arrangorRepository.getByOrganisasjonsnummer(virksomhetsnummer)
+	override fun getOrCreateArrangor(arrangor: Arrangor): Arrangor {
+		val maybeArrangor = arrangorRepository.getByOrganisasjonsnummer(arrangor.organisasjonsnummer)
```

Comment:
Ikke noe nytt egentlig ser jeg men syns det er litt snodig med en `getOrCreate` metode. Kan man ikke da tenke at det burde isåfall være `getOrCreateOrUpdate`? Ser ikke helt flyten så mulig det ikke er logisk. Men tenker på at man egentlig isåfall bare kan bruke eksisterede `upsertArrangor`, siden den returnerer jo den nye/oppdaterte arrangøren

Reply:
Skjønner hva du mener. Jeg tenker at det kan være greit å bytte ut sånn at jeg bruker upsertArrangør-metoden uansett, men siden den kun kalles hvis arrangøren ikke finnes tror jeg det blir mindre forvirrende om jeg beholder navnet :)